typespec-extension/src/operation-utils.ts

```diff
@@ -41,33 +42,33 @@ export function isKnownContentType(contentTypes: string[]): boolean {
     });
 }
 
-export function operationIsJsonMergePatch(op: HttpOperation): boolean {
-  return operationIsContentType(op, "application/merge-patch+json");
+export function sdkHttpOperationIsJsonMergePatch(op: SdkHttpOperation): boolean {
+  return sdkHttpOperationIsContentType(op, "application/merge-patch+json");
 }
 
-export function operationIsMultipart(op: HttpOperation): boolean {
-  return operationIsContentType(op, "multipart/form-data");
+export function sdkHttpOperationIsMultipart(op: SdkHttpOperation): boolean {
+  return sdkHttpOperationIsContentType(op, "multipart/form-data");
 }
 
-function operationIsContentType(op: HttpOperation, contentType: string): boolean {
-  for (const param of op.parameters.parameters) {
-    if (param.type === "header" && param.name.toLowerCase() === CONTENT_TYPE_KEY) {
-      if (param.param.type.kind === "String" && param.param.type.value === contentType) {
+function sdkHttpOperationIsContentType(op: SdkHttpOperation, contentType: string): boolean {
+  for (const param of op.parameters) {
+    if (param.kind === "header" && param.serializedName.toLowerCase() === CONTENT_TYPE_KEY) {
+      if (param.type.kind === "constant" && param.type.value === contentType) {
```

Comment:
Same ask on naming of these. If you do not need to change the function name, keep it same.